frontend/src/components/ContactSelector.tsx

```diff
@@ -43,6 +43,7 @@ export const ContactSelector: React.FC<Props> = ({ selected = [], contacts, onCh
     <CreatableSelect
       isMulti={!isTransfer}
       autoFocus
+      // menuIsOpen // debug
```

Comment:
remove comment

Reply:
I wanted to leave that there, because I spent 10 minutes trying to figure out how to force that damn thing open for debugging.  :)